versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx

```diff
@@ -355,15 +367,17 @@ spec:
 ```
 
 ## Deploying the application on a Single Cluster
```

Comment:
heading should be in the title case. 

Change `application` to `Application`

Reply:
updated already